app/helpers/application_helper.rb

```diff
@@ -15,11 +15,7 @@ def default_title_content
   end
 
   def active_class(name)
-    if name.find { |n| request.path.ends_with?(n) }
-      "active"
-    else
-      controller_path
-    end
+    (name.include?(controller_path) || name.include?(controller_path + '/' + action_name)) ? 'active' : controller_path
```

Comment:
@dorner not only "could" this mess us up, upon clicking through every menu entry it was in fact messing things up. This switches to look at the whole path or the path+action.